content/en/docs/duplicate-certificate-limit.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Duplicate Certificate Limit
+slug: duplicate-certificate-limit
+top_graphic: 1
+date: 2022-06-16
+lastmod: 2022-06-16
+show_lastmod: 1
+---
+
+
+# Description
+All issuance requests are subject to a *Duplicate Certificate* limit of 5 per
+week. You should receive the following error message from your ACME client when
+you’ve exceeded the Duplicate Certificate limit:
```

Comment:
I'd say "You should receive and error message like the following...", just because pieces of it (like the number of certificates, the number of hours, and ofc the domain names) might all change in the future.